### PR TITLE
react-redux redux-thunk and redux are implemented

### PR DESCRIPTION
**Pull request by AmanSharma4419**
react-redux redux-thunk and redux are implemented

### CODE REVIEW

File: client/index.js [hunk 1]
```diff
@@ -1,6 +1,12 @@
-import React from "react";
-import ReactDOM from "react-dom";
-import App from "./components/App.jsx";
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './components/App.jsx';
+import { Provider } from 'react-redux';
+import Store from './components/store/store';
```
Comment (prank7) [inline client/index.js]: `Store` should be `store`.

File: client/components/store/store.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { createStore, applyMiddleware, combineReducers } from 'redux';
+import handleReducer from '../reducers/reducers';
```
Comment (prank7) [inline client/components/store/store.js]: No thing as `handleReducer`, its `rootReducer`.

File: client/components/reducers/reducers.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+// const state = {
```
Comment (prank7) [inline client/components/reducers/reducers.js]: Why is this file commented? Will store work without reducer?

**Review by prank7 — CHANGES_REQUESTED** [review]

File: client/components/reducers/reducers.js (hunk 3 above)
Comment (BadBrahmin) [reply client/components/reducers/reducers.js]: Will be elaborated today.

File: client/components/store/store.js (hunk 2 above)
Comment (BadBrahmin) [reply client/components/store/store.js]: resolved.

**Pull request merged**